core/src/components/modal/animations/sheet.ts

```diff
@@ -12,7 +12,13 @@ export const createSheetEnterAnimation = (opts: ModalAnimationOptions) => {
    * current breakpoint, then the backdrop should be fading in.
    */
   const shouldShowBackdrop = backdropBreakpoint === undefined || backdropBreakpoint < currentBreakpoint!;
-  const initialBackdrop = shouldShowBackdrop ? `calc(var(--backdrop-opacity) * ${currentBreakpoint!})` : '0';
+
+  const initialBackdrop =
+    opts.theme === 'ionic'
```

Comment:
Let's not using theme to check. I would use `opts.initialBackdrop`. Read the comment below regarding this.